lib/datasource/docker/index.ts

```diff
@@ -87,6 +98,17 @@ async function getDockerApiTags(
   return tags;
 }
 
+function isECRMaxResultsError(err): boolean {
```

Comment:
Please move to util or common.

Reply:
Good call, thanks. Move in https://github.com/renovatebot/renovate/pull/13531/commits/eeacd45e045c6ef1717f6136db655304e0aec45e.